build_membership.py

```diff
@@ -147,6 +163,216 @@ def test_{binding}_members():
+                "QScriptContext",
+                "QScriptContextInfo",
+                "QScriptEngine",
+                "QScriptEngineAgent",
+                "QScriptString",
+                "QScriptValue",
+                "QScriptValueIterator",
+                "__doc__",
+                "__file__",
+                "__name__",
+                "__package__",
+            ],
+
+            "QtNetwork": [
+                # missing from Qt 4
+                "QIPv6Address",
+            ],
+
+            "QtPrintSupport": [
+                # PyQt4
+                "QAbstractPrintDialog",
+                "QPageSetupDialog",
+                "QPrintDialog",
+                "QPrintEngine",
+                "QPrintPreviewDialog",
+                "QPrintPreviewWidget",
+                "QPrinter",
+                "QPrinterInfo",
+            ],
+
+            "QtWidgets": [
+                # PyQt4
+                "QTileRules",
+
+                # PyQt5
+                "QGraphicsItemAnimation",
+                "QTileRules",
+
+                "qApp",  # See Issue #171
+            ],
+
+            "QtHelp": [
+                # PySide
+                "QHelpContentItem",
+                "QHelpContentModel",
+                "QHelpContentWidget",
+                "QHelpEngine",
+                "QHelpEngineCore",
+                "QHelpIndexModel",
+                "QHelpIndexWidget",
+                "QHelpSearchEngine",
+                "QHelpSearchQuery",
+                "QHelpSearchQueryWidget",
+                "QHelpSearchResultWidget",
+            ],
+
+            "QtXml": [
+                # PySide
+                "QDomAttr",
+                "QDomCDATASection",
+                "QDomCharacterData",
+                "QDomComment",
+                "QDomDocument",
+                "QDomDocumentFragment",
+                "QDomDocumentType",
+                "QDomElement",
+                "QDomEntity",
+                "QDomEntityReference",
+                "QDomImplementation",
+                "QDomNamedNodeMap",
+                "QDomNode",
+                "QDomNodeList",
+                "QDomNotation",
+                "QDomProcessingInstruction",
+                "QDomText",
+                "QXmlAttributes",
+                "QXmlContentHandler",
+                "QXmlDTDHandler",
+                "QXmlDeclHandler",
+                "QXmlDefaultHandler",
+                "QXmlEntityResolver",
+                "QXmlErrorHandler",
+                "QXmlInputSource",
+                "QXmlLexicalHandler",
+                "QXmlLocator",
+                "QXmlNamespaceSupport",
+                "QXmlParseException",
+                "QXmlReader",
+                "QXmlSimpleReader",
+            ],
+
+            "QtTest": [
+                # missing from PySide
+                "QTest",
+            ],
+        }
+    elif _pyside2_commit_date() <= datetime.datetime(2017, 8, 25):
+        exclude_dict = {}
```

Comment:
@mottosso okay so I made a fresh dictionary for this new version of PySide2. Not sure if this is how we should do this... please advise. 😀